<!-- Usage of the launch file: roslaunch sampler waypoint_sender.launch file:=mission_file.plan  -->

<launch>
	<!-- Declare the mission file as an argument -->
	<arg name="mission_file" default="None" />
  <!-- Launch waypoint sender node -->

	<node name="waypoint_sender_node" pkg="sampler" type="waypoint_sender_node_sweeping_mission.py" clear_params="true" output="screen">
    		<param name="mission_directory" value="$(find sampler)/missions/" />
    		<param name="file_name" value="$(arg mission_file)"/>
    		
	</node>
</launch>
